graphql/queryResolvers/session.test.js

```diff
@@ -5,9 +5,9 @@ import db from '../../helpers/dbload'
 const { Submission, UserLesson, Star, User } = db
 
 describe('Session resolver', () => {
-  test('should return null if req.user does not exist', async () => {
+  test('should return empty session if req.user does not exist', async () => {
     const req = {}
-    expect(await session({}, {}, { req })).toEqual(null)
+    expect(await session({}, {}, { req })).toEqual({ lessonStatus: [] })
```

Comment:
Incorrect resolver was causing problems, so I fixed it. Session can't be null it's at least `{ lessonStatus: [] }` with our types.